code/go/0chain.net/blobbercore/stats/blobberstats.go

```diff
@@ -173,6 +179,53 @@ func (bs *BlobberStats) loadDetailedStats(ctx context.Context) {
 	}
 }
 
+func (bs *BlobberStats) loadInfraStats(ctx context.Context) {
+	healthIn := ctx.Value(HealthDataKey)
+	if healthIn == nil {
+		fmt.Println("loadInfraStats err: health should not be nil")
+		return
+	}
+	health := healthIn.(string)
+	memstats := runtime.MemStats{}
+	runtime.ReadMemStats(&memstats)
+	bs.InfraStats = InfraStats{
+		CPUs:               runtime.NumCPU(),
+		NumberOfGoroutines: runtime.NumGoroutine(),
+		HeapSys:            int64(memstats.HeapSys),
+		HeapAlloc:          int64(memstats.HeapAlloc),
+		ActiveOnChain:      health,
+	}
+}
+
+func (bs *BlobberStats) loadDBStats() {
+	dbstats, err := GetDBStats()
+	if err != nil {
+		fmt.Println(err)
```

Comment:
err is unnecessary here. because err is handled by your `GetDBStats`.

Reply:
I think there are too many small methods. it is not good for understanding. for loadDbStats and GetDBstats.  it is better to merge them into one mehod `loadDBStats`.

Reply:
merged in single function